content/reporting/adoc/en/creation.adoc

```diff
@@ -787,6 +787,55 @@ At least one row definition must be added for the serial chart:
 * *Type* - row display type.
 * *Color field* - a field from which segment colors will be taken. The color value should be specified in the https://en.wikipedia.org/wiki/Web_colors[web] format. If not defined, colours will be chosen automatically.
 
+[[pivotTable_output]]
+==== Pivot Table Formatter
+
+Pivot Table output type is available if the application project includes the *charts* component. For more information in Pivot Table see the https://doc.cuba-platform.com/charts-latest/pivotTable.html[corresponding chapter] in the *Charts* manual.
```

Comment:
For more information in Pivot Table see the https://doc.cuba-platform.com/charts-latest/pivotTable.html[Charts manual].